app/model/GuardianWeeklyZones.scala

```diff
@@ -7,7 +7,6 @@ object GuardianWeeklyZones {
   private val allCountries = CountryGroup.allGroups.flatMap(_.countries).toSet
 
   val zoneACountryGroups = Set(CountryGroup.UK, CountryGroup.US)
```

Comment:
This is still being used in a couple of places, and I didn't want to rip out too much code at once!